imap_processing/cdf/tests/test_cdf_attribute_manager.py

```diff
@@ -1,58 +1,322 @@
 
-    # Expected failure: file_naming_convention is not in schema.
-    # for attr in cdf_manager.global_attributes.keys():
-    #     assert attr in cdf_manager.global_attribute_schema.keys()
-
+    # Testing information has been loaded in
+    assert cdf_manager.global_attributes["Project"] == "STP>Solar-Terrestrial Physics"
+    assert (
+        cdf_manager.global_attributes["Source_name"]
+        == "IMAP>Interstellar Mapping and Acceleration Probe"
+    )
+    assert (
+        cdf_manager.global_attributes["Discipline"]
+        == "Solar Physics>Heliospheric Physics"
+    )
     assert (
         cdf_manager.global_attributes["Mission_group"]
         == "IMAP>Interstellar Mapping and Acceleration Probe"
     )
+    assert cdf_manager.global_attributes["PI_name"] == "Dr. David J. McComas"
+    assert (
+        cdf_manager.global_attributes["PI_affiliation"]
+        == "Princeton Plasma Physics Laboratory, 100 Stellarator Road, "
+        "Princeton, NJ 08540"
+    )
+    assert (
+        cdf_manager.global_attributes["File_naming_convention"]
+        == "source_descriptor_datatype_yyyyMMdd_vNNN"
+    )
+    # The following test should fail because "DOI" is not an attribute in
+    #   imap_default_global_cdf_attrs.yaml
+    with pytest.raises(KeyError):
+        assert cdf_manager.global_attributes["DOI"] == "test"
 
-    # Load additional global attributes
-    cdf_manager.load_global_attributes("imap_mag_global_cdf_attrs.yaml")
+    # Load in different data
+    cdf_manager.source_dir = Path(__file__).parent.parent / "tests"
+    cdf_manager.load_global_attributes("imap_default_global_test_cdf_attrs.yaml")
 
+    # Testing attributes carried over
     assert (
-        cdf_manager.global_attributes["Mission_group"]
+        cdf_manager.global_attributes["File_naming_convention"]
+        == "source_descriptor_datatype_yyyyMMdd_vNNN"
+    )
+    assert (
+        cdf_manager.global_attributes["Discipline"]
+        == "Solar Physics>Heliospheric Physics"
+    )
+
+    # Testing attributes newly loaded
+    assert cdf_manager.global_attributes["Project"] == "STP>Solar-Terrestrial Physics"
+    assert (
+        cdf_manager.global_attributes["Source_name"]
         == "IMAP>Interstellar Mapping and Acceleration Probe"
     )
+    assert cdf_manager.global_attributes["Mission_group"] == "Dysfunctional Cats"
+    assert cdf_manager.global_attributes["PI_name"] == "Ana Manica"
+    assert cdf_manager.global_attributes["PI_affiliation"] == "LASP, CU"
+    assert cdf_manager.global_attributes["Data_version"] == 2
+    assert cdf_manager.global_attributes["DOI"] == "test"
 
-    assert cdf_manager.global_attributes["Descriptor"] == "MAG>Magnetometer"
+    # Testing that everything loaded into the global attrs is present in
+    #   the global attrs schema
+    for attr in cdf_manager.global_attributes.keys():
+        assert attr in cdf_manager.global_attribute_schema.keys()
```

Comment:
Here, you're actually just testing your test files, which isn't particularly helpful. If you think about it, what lines of code is this testing? You've already tested that the global attributes are loaded in correctly. So I'd remove this assert.